_data/glossary.yaml

```diff
@@ -289,4 +289,14 @@ volume: |
   independent of the container's life cycle. Docker therefore never automatically
   delete volumes when you remove a container, nor will it "garbage collect"
   volumes that are no longer referenced by a container.
-  *Also known as: data volume*
+  It is also known as *data volume.*
+
+  There are three types of volumes: *host, anonymous, and named*:
+
+  - A **host volume** lives on the Docker host's filesystem and can be accessed from within the container.
+
+  - An **anonymous volume** is useful for when you would rather have Docker handle where the files are stored.
+    It can be difficult, however, to refer to the same volume over time when it is an anonymous volumes.
```

Comment:
You should not use anonymous volumes unless you know for a fact that you don't want to reuse the volume. It's an implementation detail that anonymous volumes have randomly-generated names.